inst/rmarkdown/homepage/index.Rmd

```diff
@@ -0,0 +1,363 @@
+Dit kan worden gedaan door een gepubliceerd `sfp` protocol op te nemen als een `spp` subprotocol waarin de afwijkingen van de standaardinstellingen worden gegeven.
+
+
+### Protocolidentificatie
+
+
+De eerder uiteengezette typologie van protocollen wordt verder uitgewerkt in een zogenaamde protocolcode.
+Een **protocolcode** bestaat uit een voorvoegsel dat verwijst naar het *protocoltype* (drie tekens), een *protocolnummer* (drie cijfers) en een *taalcode* (twee tekens), alle gescheiden door een koppelteken.
+
+| type         | thema      | thema nummer | protocol-code |
+|:-------------|:-----------|:-------------|:--------------|
+| veld         | generiek   | 0            | sfp-0##-nl    |
+| veld         | water      | 1            | sfp-1##-nl    |
+| veld         | lucht      | 2            | sfp-2##-nl    |
+| veld         | bodem      | 3            | sfp-3##-nl    |
+| veld         | vegetatie  | 4            | sfp-4##-nl    |
+| veld         | soorten    | 5            | sfp-5##-nl    |
+| Instrument   |            |              | sip-###-nl    |
+| operationeel |            |              | sop-###-nl    |
+| Analyse      |            |              | sap-###-nl    |
+| Project      |            |              | spp-###-nl    |
+
+De drie cijfers van de protocolcode worden het *protocolnummer* genoemd.
+Bij themaprotocollen komt het eerste cijfer van het protocolnummer overeen met het themanummer.
+De `##` geven een oplopend nummer aan.
+Zo zal het eerste veldprotocol voor "thema water" de protocolcode `sfp-101-en` hebben.
+Het `s*p-###` deel van de protocolcode kan worden beschouwd als een code die één op één overeenkomt met de titel van het protocol (wanneer de taal buiten beschouwing wordt gelaten).
+
+De laatste twee karakters identificeren de taal waarin het protocol is geschreven.
+Dit kan het Nederlands (`nl`) of het Engels (`en`) zijn.
+
+Elk protocol heeft dus een unieke protocolcode, maar er kunnen meerdere versies van een protocol zijn.
+Om onderscheid te maken tussen verschillende versies van een protocol, hebben we een versienummer nodig.
+Het **versienummer** dat we gebruiken is van de vorm `YYYY.NN`.
+`YYYY` geeft het jaar aan waarin het protocol is uitgebracht.
+Het `NN` is een getal dat de volgorde van uitgave binnen dat jaar aangeeft (beginnend met 01).
+Het versienummer identificeert dus op unieke wijze een specifieke versie van een protocol, zelfs zonder dat naar de protocolcode hoeft te worden verwezen.
+Hier is een fictief voorbeeld:
+
+- versie `2022.01` (identificeert een eerste versie van `sfp-401-en`)
+- versie `2022.02` (identificeert een eerste versie van `sfp-001-en`)
+- versie `2022.03` (identificeert een tweede versie van `sfp-401-en`)
+
+Op de NEWS-pagina staan de versienummers en wordt aangegeven bij welke protocollen ze horen.
+De website heeft ook een overzichtspagina voor elk protocol (zoals geïdentificeerd door de protocol-code) waar alle versies van dat protocol staan.
```

Comment:
Als ik vanuit een protocol naar die overzichtspagina ga, opent die niet spontaan in een nieuwe tab, terwijl ik dit wel handig zou vinden om na het zien van de tabel even terug te kunnen kijken welke protocolcode het protocol had.  Anderzijds opent een protocol wel altijd in een nieuwe tab, als ik dit open vanuit de overzichtspagina (wat ik minder noodzakelijk vind, maar toch wel eerder handig).  Ik weet niet of het mogelijk is om ook in het protocol de link zodanig te maken, dat die in een nieuwe tab opent?

Reply:
Goed idee. Ik heb het aangepast.